### PR TITLE
Generic fix for the profile view in toolbar

### PR DESCRIPTION
**Pull request by iFlameing**
The previous fix breaks in some cases but this one is a generic one tested in firefox and chrome. The previous fix also works great in both browsers but it breaks when we have a site with a different language.

### CODE REVIEW

**Comment by tisto** [comment]
This is something that @sneridagh needs to review.

**Review by sneridagh — COMMENTED** [review]
@iFlameing @tisto what's the culprit that this PR fixes? I'm pretty sure that removing that transform is breaking something in the mobile view of the toolbar.

**Comment by iFlameing** [comment]
I have to check the mobile view

**Comment by tisto** [comment]
@sneridagh the original issue was the the only way to close the "add-content-menu" in Volto was to click below it. If you click anywhere to the right area of the menu, the add-content-menu wouldn't close.

**Comment by iFlameing** [comment]
@tisto @sneridagh It is ready for review.
Attaching demo 
ezgif com-video-to-gif (3)

**Comment by iFlameing** [comment]
Yep!!!

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
 ### Bugfix
 
+- Use Plone `I18N_LANGUAGE` cookie instead of `language` @cekk
```
Comment (tisto) [inline CHANGELOG.md:11]: @iFlameing this was already released. Pls remove the duplicate line.

**Comment by sneridagh** [comment]
I tried it out both in browser and in a real iOS simulator and is still broken:
Simulator Screen Shot - iPhone SE (2nd generation) - 2020-10-16 at 11 04 12

**Review by sneridagh — APPROVED** [review]

**Comment by iFlameing** [comment]
@tisto it is ready for merge

**Pull request merged**